ui/plugins/failure_summary/main.html

```diff
@@ -63,6 +63,16 @@
 
         </li>
 
+        <li ng-repeat="error in errors">
```

Comment:
I would put a header of some kind indicating these are log lines before this list.